src/docfx/config/AppData.cs

```diff
@@ -46,6 +49,11 @@ public static string GetCommitCachePath(string remote)
             return Path.Combine(CacheRoot, "commits", HashUtility.GetMd5Hash(remote));
         }
 
+        public static string GetCommitBuildTimePath(string remote, string branch)
+        {
+            return Path.Combine(CacheRoot, "history", $"build_history_{HashUtility.GetMd5Guid(remote)}_{HashUtility.GetMd5Guid(branch)}.json");
```

Comment:
>remote [](start = 94, length = 6)

Does this remote normalize?

Reply:
It is the value of `DOCS_REPOSITORY_URL` or the raw value got from the current `git remote show`. docfx does not normalize it. If build service ensures `DOCS_REPOSITORY_URL` and git remote URL is normalized then this is correct.

Reply:
we do remove `.git` extension from `git remote show`